content/docs/welcome-guide/requirements.md

```diff
@@ -119,3 +115,98 @@ If the current CMake version was not returned because CMake cannot be found, loc
 1. Choose **OK** to save your changes and close the windows that were opened.
 
 1. Verify that `cmake` is on the system path by opening a new command line window and running `cmake --version` again.
+
+## macOS
+
+Support for developing on macOS is in an experimental stage. At a minimum, you need macOS Catalina, Intel x86_64, XCode 11, and a Metal-compatible video card that meets the hardware requirements above.
+
+## Linux 
+
+The primary Linux distribution for using the O3DE editor is Ubuntu 20.04.3 LTS. The following instructions will described how to retrieve and install the required software packages through Ubuntu's **apt-get** command-line utility.
+
+
+### CMake
+
+As with the other operating systems, [CMake {{< versions/cmake >}} or later](https://cmake.org/download/) is required to configure and build O3DE projects. We strongly recommend that you install the **Latest Release** from the CMake rather than the default one provided by your current Linux distribution. If there is already a cmake installed, but does not match the minimum version, you will need to remove it with the following command.
```

Comment:
```suggestion
As with the other operating systems, [CMake {{< versions/cmake >}} or later](https://cmake.org/download/) is required to configure and build O3DE projects. We strongly recommend that you install the **Latest Release** of CMake rather than the default one provided by your current Linux distribution. If CMake is already installed, but does not match the minimum version, you will need to remove it with the following command.
```